index.js

```diff
@@ -64,10 +66,8 @@ module.exports = {
     ].filter(Boolean));
   },
 
-  treeForPublic() {
-    let parentAddon = this.parent.findAddonByName(this.parent.name());
-    if (!parentAddon) { return; }
-
+  treeForPublic() { 
+    let parentAddon = this.parent.treePaths ? this.parent : this;
```

Comment:
Ha! Yes, ironically I forgot to move that into my devDeps. Whoops! Reverting the index.js file now. Good catch 💯